libs/blocks/merch-card/merch-card.js

```diff
@@ -325,10 +346,11 @@ const init = async (el) => {
     }
     footerRows = getMiniCompareChartFooterRows(el);
   }
-  const images = el.querySelectorAll('picture');
+  const allPictures = el.querySelectorAll('picture');
+  const pictures = Array.from(allPictures).filter((picture) => !picture.closest('.mnemonic-list'));
```

Comment:
please rather use the cleaner (and slightly more efficient)
`const pictures = mnemonicList.querySelectoryAll('picture')`

Reply:
This is to first get all picture els and filter those inside mnemonic list so that we only get merch-icons. `mnemonicList` gets initialized later in the `parseContent()` since we decorate card elements in cascading order.

Reply:
oups, indeed, sorry about that